test/com/google/javascript/jscomp/WhitespaceWrapGoogModulesTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.javascript.jscomp;
+
+import com.google.javascript.jscomp.CompilerOptions.LanguageMode;
+
+public class WhitespaceWrapGoogModulesTest extends CompilerTestCase {
+
+  private LanguageMode languageOut;
+
+  @Override
+  public void setUp() {
+    setAcceptedLanguage(LanguageMode.ECMASCRIPT6);
+    languageOut = LanguageMode.ECMASCRIPT6;
+    disableTypeCheck();
+    enableLineNumberCheck(false);
+    enableCompareAsTree(false); // otherwise the "use strict" pops to the top and breaks the test
```

Comment:
I'm trying to assert that the output is identical to ClosureBundler, i.e. wrapping the entire script in:
``` javascript
goog.loadModule(function(exports) {'use strict';
...
;return exports;});
```
When I remove ```enableCompareAsTree(false);``` the unit fails because the expected output has been parsed and reformatted with "use strict" removed or moved elsewhere.